generators/docker/templates/src/main/docker/keycloak.yml.ejs

```diff
@@ -37,7 +37,7 @@ services:
       - 127.0.0.1:9080:9080
       - 127.0.0.1:9443:9443
     healthcheck:
-      test: ['CMD', 'curl', '-f', 'http://localhost:9080/realms/jhipster']
+      test: ['CMD', 'nc', '-z', '-w2', 'localhost', '9080']
```

Comment:
I believe we need to add `KC_HEALTH_ENABLED=true` to the environment and use the following for the test:

```
test: "bash /opt/keycloak/health-check.sh"
```

From https://twitter.com/ubaid4j/status/1666489377537822721

Reply:
The `health-check.sh` script also needs to be added.

https://github.com/ubaid4j/api-gateway-with-multiple-oidc-clients/blob/main/src/main/docker/keycloak/health-check.sh

Reply:
@BrayanMnz I tried to update this PR and push the changes to your branch, but I get a permission denied error. Maybe you can temporarily add me to your fork?

Reply:
done!